docs/getting-started-guides/logging.md

```diff
@@ -167,7 +167,7 @@ Here is some sample output:
 
 ![BigQuery](bigquery-logging.png)
 
-We could also fetch the logs from Google Cloud Storage buckets to our desktop or laptop and then search them locally. The following command fetches logs for the counter pod running in a cluster which is itself in a GCE project called `myproject`. Only logs for the date 2015-06-11 are fetched.
+We could also fetch the logs from Google Cloud Storage buckets to our desktop or laptop and then search them locally. The following command fetches logs for the counter pod running in a cluster which is itself in a Compute Engine project called `myproject`. Only logs for the date 2015-06-11 are fetched.
```

Comment:
so projects are in both GCE and GKE, so maybe they're Google Cloud Platform (GCP) projects? Would this introduce unnecessary confusion?